src/stream_base-inl.h

```diff
@@ -32,9 +32,10 @@ StreamReq* StreamReq::FromObject(v8::Local<v8::Object> req_wrap_obj) {
 }
 
 void StreamReq::Dispose() {
-  std::unique_ptr<StreamReq> ptr(this);
+  BaseObjectPtr<AsyncWrap> destroy_me{GetAsyncWrap()};
```

Comment:
can changes to core streams be pulled into a standalone PR? this affects everything, not just QUIC